components/explore/ExploreSort.vue

```diff
@@ -64,6 +64,8 @@ const options = computed(() => {
     ? NFT_SQUID_SORT_CONDITION_LIST
     : NFT_SQUID_SORT_COLLECTIONS
 })
+const label = computed(() => (isMobileDevice ? '' : 'Sort By'))
+const icon = computed(() => (isMobileDevice ? 'filter' : ''))
```

Comment:
As I said, I don't like the use of isMobileDevice beacsue it doesn't actually check screen size
So you could have unexpected results for bigger screen that are touch

I  would write the button twice once with is-hidden-mobile and once with is-hidden-tablet classes to toggle the views

If you don't like that approach then I would do something like this 

```
Const width =ref(window.innerWidth) 
onMounted(() => { 
   window.addEventListener('resize', () => { 
     width.value = window.innerWidth 
   }) 
 }) 
  
  
 const isMobile = computed(() => width.value <= 768) 
 ```

Reply:
> I would write the button twice, once with is-hidden-mobile and once with is-hidden-tablet classes to toggle the views

^ this